docs/_ext/single_sourced_data.py

```diff
@@ -45,8 +45,8 @@
 ]
 
 
-DOCS_DIR = Path(__file__).parents[1].resolve()
-PROJECT_DIR = DOCS_DIR.parent
+DOCS_DIR = Path(__file__).resolve().parents[1]
```

Comment:
This feels a bit unnatural.
It could also be
```suggestion
DOCS_DIR = Path(__file__, '..').resolve()
```
I guess.